### DISCUSSION TITLE
Cables Export

### DISCUSSION THREAD

**Discussion by romarioZ1000** [netbox-community/netbox#6027] [Q&A] [answered]
Hello everyone! Help me to understand.
I have in dcim/cables/ list cables.
I need Export my list not default method.
I create new template: 

{% for cables in queryset %}
Label  : {{ label }}
Site A: {{ termination_a }}
Site B: {{ termination_b }}
{% endfor %}

after that i get an empty list

![1](https://user-images.githubusercontent.com/67051805/111914804-426dc500-8a7c-11eb-94f6-176603c1a011.png)
![2](https://user-images.githubusercontent.com/67051805/111914809-47327900-8a7c-11eb-92be-5c25aa407453.png)
![3](https://user-images.githubusercontent.com/67051805/111914815-4a2d6980-8a7c-11eb-84d6-e5ebc9bd4479.png)

and same problem with rear_port and front_port.

anybody have full list with queryset in netbox???

Thx for any answers!

**Comment by rodvand** [comment, marked as answer]
You're starting the loop by saying `for cables in queryset` this means each entry will be within the `cables` variable. So if you are to reference anything within the loop you need to do `{{ cables.label }}` etc.

**Reply by romarioZ1000** [reply]
Thank you! It`s work

{% for cables in queryset %}
Label  : {{ cables.label }}
Site A: {{ cables.termination_a }}
Site B: {{ cables.termination_b }}
{% endfor %}



Please tell me, where can I find out the entire queryset list? For example, are these queries from the postgres database? not find the entire list of variables.

**Reply by rodvand** [reply]
Go to your-netbox-url/api/dcim/cables/ to get an overview of the available variables for an entry.

**Reply by candlerb** [reply]
Export templates use the Django models, rather than the REST API, and the two don't always match.

One way to get model documentation is to use the [netbox shell](https://netbox.readthedocs.io/en/stable/administration/netbox-shell/):

```
root@netbox:~# cd /opt/netbox/
root@netbox:/opt/netbox# . venv/bin/activate
(venv) root@netbox:/opt/netbox# python3 netbox/manage.py nbshell
### NetBox interactive shell (netbox)
### Python 3.6.9 | Django 3.1.7 | NetBox 2.10.6
### lsmodels() will show available models. Use help(<model>) for more info.
>>> help(Cable)
```

**Reply by romarioZ1000** [reply]
Thanks a lot!!!
ALL working and I All Understood!!
